### PR TITLE
Stephen/testnet configs

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

File: src/pages/validator/amplifier/verifier-onboarding.mdx [hunk 1]
```diff
@@ -285,29 +285,29 @@ The following are an example `config.toml` files:
    type="MultisigSigner"
 
    [[handlers]]
-   chain_name="avalanche"
-   chain_rpc_url="https://avalanche-fuji-c-chain-rpc.publicnode.com"
-   cosmwasm_contract="axelar1ty7mx0cllgz8xuvhn2j7e3qy8999ssgmjtktyqetl335em0g88lq6rjhl2"
+   chain_name="ethereum-sepolia"
+   chain_rpc_url="https://rpc.ankr.com/eth_sepolia"
```
Comment (benjamin852) [inline src/pages/validator/amplifier/verifier-onboarding.mdx:289]: i've had some not so great experiences with ankr maybe suggest this endpoint?
https://ethereum-sepolia-rpc.publicnode.com

File: src/pages/validator/amplifier/verifier-onboarding.mdx [hunk 2]
```diff
@@ -338,28 +338,56 @@ The following are an example `config.toml` files:
...
+
+   [[handlers]]
+   chain_name="test-avalanche"
+   chain_rpc_url="https://api.avax-test.network/ext/bc/C/rpc"
+   cosmwasm_contract="axelar1hupk5du59cgu4ps5s637rhakwsq0060ycdp57j2ccevna7wqqzrqnfrr0p"
+   chain_finalization="ConfirmationHeight"
+   type="EvmVerifierSetVerifier"
+
+   # For each supported chain
+   #[[handlers]]
```
Comment (benjamin852) [inline src/pages/validator/amplifier/verifier-onboarding.mdx:373]: can we remove this commented code?

**Review by Olanetsoft — APPROVED** [review]

File: src/pages/validator/amplifier/verifier-onboarding.mdx (hunk 2 above)
Comment (StephenFluin) [reply src/pages/validator/amplifier/verifier-onboarding.mdx:373]: This isn't commented code, this is the template verifiers should  use for each chain they want to add.

File: src/pages/validator/amplifier/verifier-onboarding.mdx (hunk 1 above)
Comment (StephenFluin) [reply src/pages/validator/amplifier/verifier-onboarding.mdx:289]: This is coming from our canonical data set: https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-verifiers.json#L188

**Pull request merged**